tests/pages/fareType.test.tsx

```diff
@@ -52,13 +57,15 @@ describe('pages', () => {
             });
 
             it('should return expected props when a the user logged in is not a scheme operator', async () => {
-                const expectedProps = { props: { operatorName: expect.any(String), errors: [], csrfToken: '' } };
+                const expectedProps = {
+                    props: { operatorName: expect.any(String), schemeOp: false, errors: [], csrfToken: '' },
+                };
                 const mockContext = getMockContext();
                 const actualProps = await getServerSideProps(mockContext);
                 expect(actualProps).toEqual(expectedProps);
             });
 
-            it('should redirect to /passengerType when the user logged in is a scheme operator', async () => {
+            it('should not redirect to /passengerType when the user logged in is a scheme operator', async () => {
```

Comment:
Could we just remove this test? I'm thinking we probably wouldn't have written this test if this functionality had never been there?

Reply:
Yeah, I considered that at first. But then I figured that its something we used to do, and actively don't want to do now - and if we didnt have this test, then it might relapse.